caffe2/CMakeLists.txt

```diff
@@ -398,6 +398,7 @@ if(NOT INTERN_BUILD_MOBILE OR NOT BUILD_CAFFE2_MOBILE)
     "${TORCH_SRC_DIR}/csrc/autograd/generated/python_nn_functions.cpp"
     "${TORCH_SRC_DIR}/csrc/autograd/generated/python_fft_functions.cpp"
     "${TORCH_SRC_DIR}/csrc/autograd/generated/python_linalg_functions.cpp"
+    "${TORCH_SRC_DIR}/csrc/autograd/generated/python_nested_functions.cpp"
```

Comment:
Does this have internal impacts that need to be replicated in bzl files?

Reply:
Good catch, I probably need to import this and make those additions.